src/style.css

```
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url(images/star-active.svg);
+}
```

combine all selectors together like this
```suggestion
.stars--0 .stars__star:nth-child(-n),
.stars--1 .stars__star:nth-child(-n+1) {
  background-image: url(images/star-active.svg);
}
```